internal/state/mailbox_fetch.go

```diff
@@ -78,6 +81,7 @@ func (m *Mailbox) Fetch(ctx context.Context, cmd *command.Fetch, ch chan respons
 			operations = append(operations, fetchBodyStructure)
 		case *command.FetchAttributeBodySection:
 			needsLiteral = true
+			isBodyFetch = true
```

Comment:
I think it would be better if we loop over items and set `isBodyFetch=true` only if there are no items or at least one item is non-header :thinking:

Reply:
checked main email clients, there might be double counts but logs will be silent eventually.